doc/mcuboot/index-ncs.rst

```diff
@@ -0,0 +1,68 @@
+.. _mcuboot_index_ncs:
+
+MCUboot
+#######
+
+.. contents::
+   :local:
+   :depth: 2
+
+MCUboot is a secure bootloader for microcontrollers.
+
+Overview
+********
+
+The goal of MCUboot is to define a common infrastructure for the bootloader, the system flash layout on microcontroller systems, and to provide a secure bootloader that enables easy software upgrade.
+
+MCUboot is operating-system and hardware independent, and relies on hardware porting layers from the operating system it works with.
+Currently MCUboot works with the various operating systems and SDKs like `Zephyr`_ and nRF Connect SDK.
+
+MCUboot is an open governance project.
+See the `membership list <https://github.com/mcu-tools/mcuboot/wiki/Members>`_ for current members, and the `project charter <https://github.com/mcu-tools/mcuboot/wiki/MCUboot-Project-Charter>`_ for more details.
+
+Content
+*******
+
```

Comment:
Add line of introduction, as we're not using the "Subpages" here?
```suggestion
MCUBoot documentation is composed of the following pages:
```

Reply:
Ok, fixed